qemu/tests/qemu_guest_agent_update.py

```diff
@@ -0,0 +1,151 @@
+import logging
+import time
+import re
+
+from avocado.utils import process
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+from virttest import data_dir
+
+from qemu.tests.qemu_guest_agent import QemuGuestAgentBasicCheckWin
+
+
+class QemuGuestAgentUpdateTest(QemuGuestAgentBasicCheckWin):
+
+    @error_context.context_aware
+    def gagent_check_pkg_update(self, test, params, env):
+
```

Comment:
Pls add description and params explanation.

Reply:
Modified.pls have a check again.